code/datums/diseases/advance/symptoms/sensory.dm

```diff
@@ -80,30 +80,25 @@
 	if(!..())
 		return
 	var/mob/living/M = A.affected_mob
-	var/obj/item/organ/eyes/eyes = M.getorganslot(ORGAN_SLOT_EYES)
-	if (!eyes)
-		return
 	switch(A.stage)
 		if(4, 5)
-			M.restoreEars()
-
+			M.restoreEars() //this is mostly just copy+pasted from oculine and inacusiate
+			M.adjust_blindness(-2)
+			M.adjust_blurriness(-2)
+			var/obj/item/organ/eyes/eyes = M.getorganslot(ORGAN_SLOT_EYES)
+			if(!eyes)
+				return
+			eyes.applyOrganDamage(-2)
 			if(HAS_TRAIT_FROM(M, TRAIT_BLIND, EYE_DAMAGE))
 				if(prob(20))
-					to_chat(M, "<span class='notice'>Your vision slowly returns...</span>")
+					to_chat(M, "<span class='warning'>Your vision slowly returns...</span>")
```

Comment:
We usually use `warning` for failed input and the likes. Sure `notice` isn't better here?

Reply:
it's used for "important" messages as well fairly frequently